zaza/openstack/utilities/openstack.py

```diff
@@ -2444,6 +2444,45 @@ def upload_image_to_glance(glance, local_path, image_name, disk_format='qcow2',
     return image
 
 
+def import_image_to_glance(glance, local_path, image_name,
+                           disk_format='qcow2', visibility='public',
+                           container_format='bare'):
+    """Creates, stages and import an image.
```

Comment:
I'm not an Glance expert: can you explain in the docstring a bit the difference with `upload_image_to_glance()`? What's the difference between "uploading" and "staging/importing" an image?